2022q4/riscv.adoc

```diff
@@ -0,0 +1,42 @@
+=== FreeBSD/riscv64 Improvements
+
+Links: +
+link:https://wiki.freebsd.org/riscv[Wiki Homepage] URL: link:https://wiki.freebsd.org/riscv[https://wiki.freebsd.org/riscv] +
+
+Contact: Mitchell Horne <mhorne@FreeBSD.org>
+
+The FreeBSD/RISC-V project is providing support for running FreeBSD on the
+[RISC-V Instruction Set Architecture](https://riscv.org/).
```

Comment:
`link:https://riscv.org/[RISC-V Instruction Set Architecture]` for consistency. (Your markup format may work, though. I haven't tried it.)

Reply:
> … (Your markup format may work, though. I haven't tried it.)

I think, not valid in the AsciiDoc context. Viewed in GitHub: <https://github.com/mhorne/freebsd-quarterly/blob/main/2022q4/riscv.adoc#freebsdriscv64-improvements>. 

Incidentally, from <https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#links>: 

>> … The `link:` macro prefix is not required when the target starts with a URL scheme like `https:`. …